generators/app/templates/_package.json

```diff
@@ -0,0 +1,117 @@
+{
+  "name": "<%= projectNameSlugified %>",
+  "title": "<%= projectName %>",
+  "description": "<%= projectDescription %>",
+  "version": "0.0.0",
+  "repository": {
+    "type": "git",
+    "url": "<%= repoUrl %>"
+  },
+  "dependencies": {
+    <% if (includeAphrodite && !includeRadium) {%>
+      "aphrodite": "^1.2.3",<%
+    }
+
+    if (!includeAphrodite && includeRadium) {%>
+    "radium": "^0.19.4",<%
+    }
+
+    if (includeInext) {%>
+    "i18next": "^9.0.0",<%
+    }
+
+    if (includeMobileDetect) {%>
+    "mobile-detect": "^1.3.7",<%
+    }
+
+    if (includeMoment) {%>
+    "moment": "^2.18.1",<%
+    }
+
+    if (includeNukaCarousel) {%>
+    "nuka-carousel": "^2.3.0",<%
+    }
+
+    if (includeNumeral) {%>
+    "numeral": "^2.0.6",<%
+    }
+
+    if (includePostcss) {%>
```

Comment:
we should add this by default, I would also add with it the autoprefixer(https://github.com/postcss/autoprefixer) and the sprites(https://github.com/2createStudio/postcss-sprites) plugins